src/vpn-api/utils/clients.ts

```diff
@@ -83,9 +83,7 @@ const setDeviceState = (() => {
 					return targetState;
 				})
 				.catch(err => {
-					captureException(err, 'Error updating state', {
-						user: { uuid },
-					});
+					captureException(err, { tags: { uuid } });
```

Comment:
Didn't sentry treat the previous `user` property specially?  Is it worth keeping the new version using the `user` tracking stuff

Reply:
I'm not sure special sentry treats it versus other tags, but it does specifically limit the keys you can use to `id`, `username`, `email`, and `ip_address`.